.bazelrc

```diff
@@ -0,0 +1,14 @@
+build --enable_platform_specific_config
+
+build:linux --cxxopt=-std=c++17
+# See https://bytecodealliance.github.io/wasmtime/c-api/
+build:linux --linkopt=-lm
+build:linux --linkopt=-lpthread
+build:linux --linkopt=-ldl
```

Comment:
This `.bazelrc` only applies to builds in this workspace.

Shouldn't those `linkopts` be a part of `wasmtime_lib`, so that other applications that link against it could pick up them up? But even then, I think the embedding binary would need to specify them as well...

Reply:
Actually, let's deal with this once it becomes a problem.

Reply:
yeah, thanks for pointing this out. I will keep that in mind.